widget/embedded/src/components/SwapDetailsModal/SwapDetailsCompleteModal.tsx

```diff
@@ -74,19 +77,27 @@ export function SwapDetailsCompleteModal(props: CompleteModalPropTypes) {
       )}
       <Divider size={32} />
       {status === 'success' && (
-        <Button
-          id="widget-swap-details-modal-done-btn"
-          variant="contained"
-          type="primary"
-          size="large"
-          onClick={() => {
-            const home = '../../';
-            navigate(home);
-          }}>
-          <Typography variant="title" size="medium" color="neutral100">
-            {i18n.t('Done')}
-          </Typography>
-        </Button>
+        <>
+          {showProfileBanner && (
+            <div>
```

Comment:
You can replace the div element with a React fragment.

Reply:
Fixed